tasks/sysctl.yml

```diff
@@ -11,4 +11,5 @@
   with_dict: "{{ openio_sds_sysctl_entries }}"
   when: openio_sds_sysctl_managed
         and ansible_virtualization_type != 'docker'
```

Comment:
You could split the "when:" condition into a two elements list removing the "and" while you're in there